.github/workflows/deploy-gui.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and upload GUI packages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish-linux-gui:
+    runs-on: ubuntu-20.04
+    env:
+      DISPLAY: ":99.0"
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup
+        run: |
+          sudo apt-get update
+          sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 libegl-dev -y
+          /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX
```

Comment:
What does this do?

Reply:
Good point, this is not needed here anymore because the tests aren't running in the deploy workflow. I copied/pasted this from the test workflow when I moved the build over here. I believe it's there so that the tests can run in a sort of mock headless mode, I will remove it though.